### ISSUE TITLE
Implement Dataset loading

### ISSUE THREAD

**Issue by Supergecki** [Supergecki/Scientific-Computing-Project-WiSe2025_26-CIFAR-10-cGAN#1]
In `data/dataloader.py`, importing of the CIFAR-10 Dataset should be implemented.

- Use the datasets from the `torchvision` module
- Load the train_dataset and test_dataset separately (by setting the `train=True/False` parameter)
- Use `download=True` as well as the `data` directory as `root`
- Use the data transforms returned by `get_train_transform()` and `get_test_transform()`, which will be implemented in a separate Issue

Put this into a `load_data()` function that returns the dataloaders, where `batch_size` is given as an argument. Shuffle training, but not test data.
